doc/ru/Makefile.am

```diff
@@ -0,0 +1,25 @@
+#
+# NetLabel Documentation Makefile
+#
+# Author: Paul Moore <paul@paul-moore.com>
```

Comment:
I'm pretty sure you want your name here and not mine :)

I understand that this is a pretty simple cut-and-paste addition, so if you want you can always mention that this Makefile was based on doc/Makefile.am.

Reply:
Sure, changed this. Please let me know if it is still needed to further change it.

Reply:
Well, you'll need to update the PR (I can't see your changes), but see my comments about authorship and sign-offs in the main PR comment thread first.

Reply:
If you open `Commits` tab or `Files changed` tab you'll see updated files.

Reply:
Ah thanks, I looked but for some reason missed the date change when I was looking.